stable/openvpn/templates/config-openvpn.yaml

```diff
@@ -124,6 +124,10 @@ data:
       }
 
       /etc/openvpn/setup/setup-certs.sh
+{{ if .Values.openvpn.istio.enabled }}
+      iptables -t nat -A PREROUTING -s {{ .Values.openvpn.OVPN_NETWORK }}/{{ .Values.openvpn.OVPN_SUBNET }} -i tun0 -p tcp -j REDIRECT --to-ports 15001
```

Comment:
Should the port here be static, or is it possible to parameterise this as well?

Reply:
Made it configurable. Thanks for the feedback :+1: